packages/dialect-react-sdk/hooks/useThreadMessages.ts

```diff
@@ -104,17 +103,17 @@ const useThreadMessages = ({
     }
 
     // For backends other than `DialectCloud` we return null if no remote meessages
-    if (thread?.backend !== Backend.DialectCloud && !remoteMessages) {
+    if (thread?.type !== 'dialect-cloud' && !remoteMessages) {
```

Comment:
I don't like "magic strings", I understand that it's likely to be ts protected, but why did we remove the `Backend` enum here?

Reply:
Also, would it make sense to move the `isOnChain` and `isOffChain` util function to react-sdk and export them to `react-ui` and elsewhere, since we do these ops here as well (even though they are temporary)

Reply:
oh yeah, previously this const wasn't imported from sdk, replaced it with `DIALECT_API_TYPE_DIALECT_CLOUD`

Reply:
there's actually such functions `isOnChain` and `isOffChain`, so I replaced this expressions with this functions